src/main/groovy/life/qbic/portal/sampletracking/web/controllers/SampleTrackingPortletController.groovy

```diff
@@ -43,7 +46,7 @@ class SampleTrackingPortletController implements PortletController {
 
     @Override
     void clearSelection() {
-// TODO implement
+        // This is now handled directly by the view itself.
```

Comment:
then we should get rid of this method?

Reply:
Yes I did not want to change the interfaces in this PR therefore I only added the comment.

Reply:
Please delete, then it does not get forgotten